splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/SpliceDataDictionary.java

```diff
@@ -1666,6 +1669,11 @@ public void addMinRetentionPeriodColumn(TransactionController tc) throws Standar
             DataTypeDescriptor dtd = DataTypeDescriptor.getBuiltInDataTypeDescriptor(Types.BIGINT, 1);
             tc.addColumnToConglomerate(td.getHeapConglomerateId(), colNumber, storableDV, dtd.getCollationType());
 
+            try {
+                Thread.sleep(10000);
```

Comment:
not sure how often addMinRetentionPeriodColumn is called, but to have every 10 secs delay quite expensive. Why is it needed here?